js/ui/payg.js

```diff
@@ -41,8 +41,8 @@ var SPINNER_ICON_SIZE_PIXELS = 16;
 var SPINNER_ANIMATION_DELAY_SECS = 1.0;
 var SPINNER_ANIMATION_TIME_SECS = 0.3;
 
-const NOTIFICATION_TITLE_TEXT = _("PayGo");
-const NOTIFICATION_EARLY_CODE_ENTRY_TEXT = _("Enter an unlock code to extend PayGo time before expiration.");
+const NOTIFICATION_TITLE_TEXT = _("Pay As You Go");
+const NOTIFICATION_EARLY_CODE_ENTRY_TEXT = _("Enter an unlock code to extend the time before your Pay As You Go credit expires.");
```

Comment:
```suggestion
const NOTIFICATION_EARLY_CODE_ENTRY_TEXT = _("Enter an unlock code to extend the time before your credit expires.");
```

(Do we need to repeat "Pay As You Go" in the title and the body?)